ldcbench.http-node/src/main/java/org/dice_research/ldcbench/nodes/http/simple/dump/DumpFileBuilder.java

```diff
@@ -123,10 +138,41 @@ private void streamData(OutputStream out, Lang lang) {
             datasetId++;
         }
         writerStream.finish();
+        
+        
+        LOGGER.info("writerStream: " + writerStream);
         }
     }
 
-    private void writeData(OutputStream out, Lang lang) {
+    /*
+     * Converts RDF file into HDT
+     * @param The method requires an RDF file
+     * @return The method returns the HDT file
+     */
+	private File convertToHDT(File rdfFile) throws IOException, ParserException, NotFoundException {
+		String rdfInput = rdfFile.getAbsolutePath();
+		String baseURI = resourceUriTemplates[0].split("%s")[0];
+		//format language to fit RDFNotation
+		String inputType = lang.getName().replaceAll("-", "").replace('/', '-');
```

Comment:
This parsing looks very special :confused: 
Maybe coding a simple mapping from the Lang enum to the RDFNotation enum would be easier and saver? :thinking: 
You can find the code of the RDFNodation at https://github.com/rdfhdt/hdt-java/blob/b9ecbd1272846897d1c040f66fb9230a5e7d51bb/hdt-api/src/main/java/org/rdfhdt/hdt/enums/RDFNotation.java